filter-effects/Overview.bs

```diff
@@ -3456,7 +3458,16 @@ Given two filter values representing an base value (<var>base filter list</var>)
 
 ## Accumulation ## {#accumulation}
 
-Given two filter values <var>V<sub>a</sub></var> and <var>V<sub>b</sub></var>, returns the filter value, <var>V<sub>b</sub></var>.
+Accumulation of <<filter-value-list>>s follows the same matching and extending
+rules as <a href='#interpolation-of-filters'>interpolation</a>, falling back to
+[=composite operation replace|replace=] behavior if the lists do not match.
+However instead of interpolating the matching <<filter-function>> pairs, they
+are arithmetically added together - except in the case of <<filter-function>>s
+whose initial value for interpolation is 1, which combine using one-based
+addition:
+
+  <var ignore>V<sub>result</sub></var> =
+  <var>V<sub>a</sub></var> + <var ignore>V<sub>b</sub></var> - 1
```

Comment:
For my own reference, the logic here is we adding the deltas from 1 and then add the result back to 1.

i.e.

```
result = (Va - 1) + (Vb - 1) + 1
       = Va + Vb - 1
```

([Firefox implementation](https://searchfox.org/mozilla-central/rev/d7537a9cd2974efa45141d974e5fc7c727f1a78f/servo/components/style/values/animated/mod.rs#92-99))